docs/current_docs/integrations/snippets/gitlab-kubernetes.yml

```diff
@@ -0,0 +1,19 @@
+.dagger:
+  image: alpine:latest
+  variables:
+    # assumes the Dagger Cloud token is
+    # in a masked/protected variable named DAGGER_CLOUD_TOKEN
+    # set via the GitLab UI
+    DAGGER_CLOUD_TOKEN: $DAGGER_CLOUD_TOKEN
+  before_script:
+    - apk add curl
+    - curl -sfL https://releases.dagger.io/dagger/install.sh | DAGGER_VERSION=0.11.0 BIN_DIR=/tmp sh
```

Comment:
Can we change this to `latest`? It would simplify maintenance of this page but would it work? @chrira @gerhard ?

Reply:
Done

Reply:
OK to keep as is - `latest` is fine for the purpose of this documentation. +1 to what @chrira wrote further below: not good for production, OK for documentation that does not go outdated & requires maintenance.

If you think that capturing the specific version approach in a comment is worth doing @vikram-dagger, that works!